cmd/swarm/swarm-smoke/main.go

```diff
@@ -37,15 +37,16 @@ var (
 )
 
 var (
-	allhosts  string
-	hosts     []string
-	filesize  int
-	syncDelay int
-	httpPort  int
-	wsPort    int
-	verbosity int
-	timeout   int
-	single    bool
+	allhosts          string
+	hosts             []string
+	filesize          int
+	syncDelay         int
+	httpPort          int
+	wsPort            int
+	verbosity         int
+	timeout           int
+	single            bool
+	getAllRefsTimeout int
```

Comment:
call it  `trackTimeout`

Reply:
shall we not make `syncDelay, timeout, getAllRefsTimeout` duration?

Reply:
> shall we not make syncDelay, timeout, getAllRefsTimeout duration?

I was going to do it right away, but then I started doubting. This could break some deployment scripts, so I would not do it without approval by @skylenet and/or @nonsense .

In fact, the command line args would have to change to support a string duration 
https://golang.org/pkg/time/#ParseDuration

So I actually suggest to do this as a separate PR or any future PR touching these files after we know best the consequences

Reply:
Yes, I agree on changing this to a duration on a separate PR. We'll update our helm charts accordingly.